datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/DatasetComparisonJob.scala

```diff
@@ -75,11 +77,8 @@ object DatasetComparisonJob {
           expectedExceptActual.write.format("parquet").save(s"${cliOptions.outPath}/expected_minus_actual")
           actualExceptExpected.write.format("parquet").save(s"${cliOptions.outPath}/actual_minus_expected")
       }
-      throw DatasetsDifferException(cliOptions.referenceOptions.path,
-                                    cliOptions.newOptions.path,
-                                    cliOptions.outPath,
-                                    expectedDf.count(),
-                                    actualDf.count())
+      throw DatasetsDifferException(cliOptions.referenceOptions.path, cliOptions.newOptions.path,
```

Comment:
Not sure if you seen my previous comment made on an already merged PR.
According to the general Scala code style, when you need to break a method argument list into several lines, _every argument_ should be on the line _alone_. 
```suggestion
      throw DatasetsDifferException(
        cliOptions.referenceOptions.path,
        cliOptions.newOptions.path,
        cliOptions.outPath,
        expectedDf.count(),
        actualDf.count()
      )
```

Reply:
I did see it. But I also misunderstood it. 👍